### PR TITLE
Chocolate Chip Christmas Cookies

### PR DESCRIPTION
**Pull request by laf0711**
Team: Elev8

-----BEGIN PUBLIC KEY-----
MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAvXbeNmV29SWll5rBkzH1
Bcini3bjmckBfJj7+gGt5zMP7vpLaMKyl8rIEwj/2w30SAUAysMpxwZ6DpHUynL9
E9rgC8W/LqgHe4oMh+UyKQxj36kuf8YWT0q+kE5DBaZFg8Hb5aGoxF81lz3JlRq8
tPXMYih/5ajQSXrrp1CjpUtXJAhli5ixIXCDWoYiN8AzUImZPakCCOvm8Pq26zno
jAD5GS9bAaEpTj0gzzNNTVGlyvpdeYNbohHkvYZdI9m227IlPQ020VKS9JRw4F/N
s2u7mMWuX2BMWDGjijkhevu6HBr5DTv23EjP7/jayxPZHZ4KgVqZpMWlfLZjPK7O
BQIDAQAB
-----END PUBLIC KEY-----

### CODE REVIEW

File: team_Elev8/index.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        move(scoop_cookie_dough(workarea), tray);
+    }
+
+    move(tray, oven);
+
+    sleep(600);
+
+    empty_into(tray, cookiejar); 
+
+    clean(bowl);
```
Comment (StevenVanAcker) [inline team_Elev8/index.md]: don't forget to clean bowl1 and bowl2 :D

Comment (laf0711) [reply team_Elev8/index.md]: Oh, we got distracted with eating cookies. :)

Comment (laf0711) [reply team_Elev8/index.md]: Code updated so both the bowls are clean - I knew we should have just used one bowl! :)

**Comment by StevenVanAcker** [comment]
thanks :D 

az34z2amiwu0QxgndsxkzotYCh7jg3vbaL8WNrmERDkz0S8Jg9xHwp/DrdysF9y46kuq36xnAPpy
pfhni0tBQUSTBgqoHtPOGGA4rxLFUS2WzLTgh8fIUrsVFGgvkE0XERqf6hFE0AO1a18eTY9YNUqj
8QOBrsRz7kGB9008ycbCT3a84zC3kqvFhsxokwP/0uAq/wdU5qZIwHESwvshmvgHfpAmJd+EHhln
kvlLDKdb2N0ZyhrUMzmOYQZBjBamdiYcItL1szHta8TaNYD9mnqebea0R6JhqNOUBbqC/ucQa4pF
XGeoPy+6VeiPGBAbRsvyYIUymRPor+KcZQCtiA==

ODPMLkmAknbdkWc6jZIcnY/Q1akBIF3odwl5z5WxUkzFg38ivZYFpr5UW1oAsrwTmp7XWXpqUmB1
fJkQtuFTBNpyEDgp1y5fIK+usk0VvdkM2W5j4VVQuSEkmxsB5rjHm3ySZgRMhRG4lIP7BO4vMcXJ
EqHOUMHiv9/PVS67jtpFPNkvRy8CxfqIJ+VnX/I2JSseOKG2E1gll19/RPl6hs4gnpcYF9Cq7X+z
9cOWZWtxKW/E6CvY6SKuWpteHTktaCK9fKBoV1mtFd4qxVftWScwmWfAa0u9KR0eF20DYkVARK1x
kjcuT3YAYmio8Njq+Fazprl1lIOTdVIdBZ+kmA==

**Comment by StevenVanAcker** [comment]
thanks! :D 

TGTftr2EYmR1P/DfBRcICYn20M4QLUC9lYKRJf9XQL8DSqG2u2l29GylALCHGAzsfpcZmZh7FEQh
qN6kBxneT0lLLyVI35becfHzGro5+IYm2vquKu7QvKQXbK/6Oj5zvnIxKxftNHuJ556M3rDgxmwP
ckDRoq+CdKDSFX2Py731HfdTUCII7s1DefAuqZtqNzM7s2btizaO2OTPEtCm2zlPiSSmRfrLfC3T
EU0OiuwaEN5VMkT9vkm3x5qU9YYGLaNtIw2/3wtEibfIc3k9k6G9phN8JibZ4+iUN+ov2sNLCWU/
ngOU/MMSv5ZwYyM5Ku1/zk0pv1dAYmV4JphtIQ==

**Pull request merged**